.openshift-ci/dispatch.sh

```diff
@@ -31,6 +31,9 @@ case "$ci_job" in
     gke-upgrade-tests)
         "$ROOT/.openshift-ci/gke_upgrade_test.py"
         ;;
+    shell-unit-tests)
+        make shell-unit-tests
```

Comment:
The shell tests write some output to `shell-test-output`:
https://github.com/stackrox/stackrox/blob/1d10ce0f81a639fc61a70418c42226c41a3abd6b/.circleci/config.yml#L2319

I think that can be leveraged in OpenShift CI if it has JUnit style output. If you copy that dir to $ARTIFACTS. (This is not well documented but it has been mentioned on CoreOS slack #test-platform). The caveat here that there are Junit style results in that dir.

Then those results should appear on the check link as X/X tests passed. At least that is the hope.

If that is useful, and now there is more to this 'job', can you put it in `scripts/ci/jobs`. Which is where I've started putting anything more than one liners.